modules/sd_hijack_checkpoint.py

```diff
@@ -4,16 +4,19 @@
 import ldm.modules.diffusionmodules.openaimodel
 
 
+# Setting flag=False so that torch skips checking parameters.
+# parameters checking is expensive in frequent operations.
+
 def BasicTransformerBlock_forward(self, x, context=None):
-    return checkpoint(self._forward, x, context)
+    return checkpoint(self._forward, x, context, flag=False)
```

Comment:
The `checkpoint` here is `torch.utils.checkpoint.checkpoint`, and it does not have `flag=False`. I think you confused this with `ldm.modules.diffusionmodules.util.checkpoint`. The `sd_hijack_checkpoint.py` already removed the checkpointing in ldm, but we might need to do it on sgm as well.

Reply:
As far as I understand, what you are looking for is actually 
```
ldm.modules.attention.BasicTransformerBlock.forward = ldm.modules.attention.BasicTransformerBlock._forward
```

Reply:
A closer look indicates that the checkpoint here is only called when training occurs (textual_inversion & hypernetwork) so disabling checkpoint here may be undesirable.